bindings/node.js/kzg.cxx

```diff
@@ -32,42 +32,28 @@ Napi::Value throw_invalid_argument_type(const Napi::Env env, std::string name, s
   return env.Null();
 }
 
-Napi::TypedArrayOf<uint8_t> napi_typed_array_from_bytes(uint8_t* array, size_t length, Napi::Env env) {
-  // Create std::vector<uint8_t> out of array.
-  // We allocate it on the heap to allow wrapping it up into ArrayBuffer.
-  std::unique_ptr<std::vector<uint8_t>> vector =
-      std::make_unique<std::vector<uint8_t>>(length, 0);
-
-  for (size_t i = 0; i < length; ++i) {
-    (*vector)[i] = array[i];
+inline uint8_t *get_bytes(
```

Comment:
Maybe a basic small docstring here would be helpful for future rereference.

Reply:
updated